src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,16 +2,24 @@
 
 ## Unreleased
 
-* Native AOT compatibility.
-  ([#1666](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1666))
-
 * Fix a bug in `GenevaMetricExporter` where the `MetricEtwDataTransport` singleton
   is disposed.
   ([#1537](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1537))
 
 * Update OpenTelemetry SDK version to `1.8.1`.
   ([#1668](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1668))
 
+* Add OTLP protobuf encoding support for metric exporter in Windows
+  environment. Use `PrivatePreviewEnableOtlpProtobufEncoding=true` in the
+  connection string to opt-in.
+  ([#1596](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1596))
+  ([#1626](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1626))
+  ([#1629](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1629))
+  ([#1634](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1634))
```

Comment:
I don't know what's the intended format here, maybe:

```suggestion
  ([#1596](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1596),
   [#1626](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1626),
   [#1629](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1629),
   [#1634](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1634))
```

Reply:
This is much clear. Thanks!